shop-data/src/main/java/com/maxshop/di/ApiModule.kt

```diff
@@ -0,0 +1,14 @@
+package com.maxshop.di
+
+import com.maxshop.api.CategoryApi
+import dagger.Module
+import dagger.Provides
+import retrofit2.Retrofit
+
+@Module
+internal class ApiModule {
+    @Provides
+    fun getCategoryApi(retrofit: Retrofit): CategoryApi {
```

Comment:
Convert it to expression body. I mean `fun () = retrofit....`

Reply:
fixed